Core/Core/RichContentEditor/RichContentEditor.js

```diff
@@ -237,31 +236,37 @@ window.addEventListener('touchend', e => {
             img.classList.remove('editor-active')
         }
     }
-    if (e.target.nodeName.toLowerCase() === 'a') {
-        editor.currentEditingLink = e.target
-        webkit.messageHandlers.linkTap.postMessage({
-            text: e.target.textContent,
-            href: e.target.href
-        })
+    if (editor.currentEditingLink) {
```

Comment:
postState already set currentEditingLink, and the native side already received the text & url.